dev-client/src/constants.ts

```diff
@@ -35,3 +35,5 @@ export const PROJECT_DEFAULT_MEASUREMENT_UNITS: MeasurementUnit = 'METRIC';
 export const GEOSPATIAL_CONTEXT_USER_DISTANCE_CACHE = 5;
 export const SITE_NOTE_MIN_LENGTH = 3;
 export const LOCALE = 'en-US';
+export const SWITCH_PADDING = 2;
+export const SWITCH_VERTICAL_PADDING = 1;
```

Comment:
It's a bid of a misnamed constant. It's used for switches and checkboxes and radio buttons. Ideally this would be something we could specify in the theme as a default component value. We should revisit that after we eliminate nativebase.

Reply:
Ah, I see, it's not just "switch" components. 
I'm not sure the best way to put it in theme.ts, but maybe could put it under `space` after `xl: 36,` as like `togglePadding` or something?